src/Adapters/Schema/InMemorySchemaCache.js

```diff
@@ -0,0 +1,42 @@
+import type { Schema } from '../../Controllers/types';
+import { SchemaData } from '../../Controllers/SchemaController';
+import SchemaCacheAdapter from './SchemaCacheAdapter';
+
+export default class InMemorySchemaCache extends SchemaCacheAdapter {
+  dataProvider: () => Promise<{ allClasses: Array<Schema>, schemaData: SchemaData }>;
```

Comment:
this is a JS file, so I think you have to remove the type annotations

Reply:
not in parse-server. There are no ts file but there are types :)